src/RPCServiceFunctions.ts

```diff
@@ -106,58 +168,49 @@ export interface RPCServiceFunctions {
-    listAssetIndexAllocations(address: string): JsonRpcCall<AssetAllocation[]>;
-    listAssetIndexAssets(address: string): JsonRpcCall<Asset[]>;
+    getNotarySighash(hex: string, assetGuid: string): JsonRpcCall<any>;
     listAssets(count?: number, from?: number, options?: ListAssetOptions): JsonRpcCall<any>;
-    masternode(command: string): JsonRpcCall<any>;
-    masternodeBroadcast(command: string): JsonRpcCall<any>;
-    masternodeList(mode?: string, filter?: string): JsonRpcCall<any>;
     mnSync(command: string): JsonRpcCall<any>;
-    sentinelPing(version: number): JsonRpcCall<any>;
     spork(command: string): JsonRpcCall<any>;
-    syscoinBurn(fundingAddress: string, amount: number, ethAddress: string): JsonRpcCall<any>;
+    syscoinCheckMint(txid: string): JsonRpcCall<any>;
+    syscoinClearEthHeaders(): JsonRpcCall<any>;
     syscoinDecodeRawTransaction(hexString: number): JsonRpcCall<any>;
     syscoinGetSpvProof(txid: string, blockHash?: string): JsonRpcCall<any>;
-    syscoinMint(address: string, amount: number, blockNum: number, txHex: string, txRootHex: string, txMerkleProofHex: string, witness?: string): JsonRpcCall<any>;
+    syscoinGetTxRoots(height: number): JsonRpcCall<any>;
     syscoinSetEthHeaders(headers: EthHeaders): JsonRpcCall<any>;
     syscoinSetEthStatus(syncStatus: string, highestBlock: number): JsonRpcCall<any>;
     syscoinStartGeth(): JsonRpcCall<any>;
     syscoinStopGeth(): JsonRpcCall<any>;
-    syscoinTxFund(hexString: string, address: string, outputIndex: number): JsonRpcCall<any>;
-    tpsTestAdd(startTime, rawTxs?: Array<TpsRawTx>): JsonRpcCall<any>;
-    tpsTestInfo(): JsonRpcCall<any>;
-    tpsTestSetEnabled(enabled: number): JsonRpcCall<any>;
-    voteRaw(mnTxHash: string, mnTxIndex: number, govHash: string, votSignal: string, vote: string, time: number, voteSig: string): JsonRpcCall<any>;
+
+
+    // == Syscoin Wallet==
+    assetAllocationBurn(assetGuid: string, amount: number, ethAddress: string): JsonRpcCall<any>;
+    assetAllocationMint(assetGuid: string, address: string, amount: number, blockNum: number, bridgeTransferId: number, txHex: string, txRootHex: string, merkleProofHex: string, merkleProofPathHex: string, receiptHex: string, receiptRootHex: string, receiptMerkleProofHex: string, auxFeeTest?: boolean): JsonRpcCall<any>;
+    assetAllocationSend(assetGuid: string, addressTo: string, amount: number, replaceable?: boolean): JsonRpcCall<HexResponse>;
+    assetAllocationSendMany(assetGuid: string, addressFrom: string, amounts: AssetAllocationAmount[]): JsonRpcCall<HexResponse>;
+    assetNew(fundingAmount: number, symbol: string, public_value: string, contract: string, precision: number, max_supply: number, update_flags?: number,  notaryAddress?: string, notaryDetails?: any, auxfeesDetails?: any): JsonRpcCall<AssetNewResponse>;
+    assetSend(assetGuid: string, addressTo: string, amount: number, NFTID?: string): JsonRpcCall<HexResponse>;
```

Comment:
return RPCHelpMan{"assetsend",
    "\nSend an asset you own to another address.\n",
    {
        {"asset_guid", RPCArg::Type::STR, RPCArg::Optional::NO, "The asset guid."},
        {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "The address to send the asset to (creates an asset allocation)."},
        {"amount", RPCArg::Type::AMOUNT, RPCArg::Optional::NO, "Amount of asset to send."},
        {"sys_amount", RPCArg::Type::AMOUNT, RPCArg::Optional::OMITTED, "Amount of syscoin to send."},
        {"NFTID", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "Optional NFT ID to send"},
    },